packages/connect/src/data/DataManager.ts

```diff
@@ -38,6 +39,8 @@ export class DataManager {
         if (!this.settings.trustedHost && !whitelist) {
             this.settings.debug = false;
         }
+
+        this.settings.phishingDomain = isPhishingDomain(this.settings.origin || '') || true; // testing;
```

Comment:
`|| true` should be removed, right?

Reply:
Yes, comment on the end of the line (`// testing;`) is trying to make this obvious. Without this, there is no easy way how to test it